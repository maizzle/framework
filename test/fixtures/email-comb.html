<!DOCTYPE html>
<html>
<head>
  <if condition="page.css">
    <style>
      {{{ page.css }}}

      /*
        @tab Page
        @section Body Background
        @tip Set the background colour for the email body.
      */
      table {
        /*@editable*/background-color: #EEEEEE !important;
      }
    </style>
  </if>
</head>
<body>
  <!-- An HTML comment to preserve -->
  <table class="w-full">
    <tr>
      <td class="inexistent preserve-it text-center">Test</td>
    </tr>
  </table>
</body>
</html>
