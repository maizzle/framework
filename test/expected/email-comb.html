<!DOCTYPE html>
<html>
<head>
    <style>.w-full {
  width: 100% !important
}
.text-center {
  text-align: center !important
}
      /*
        @tab Page
        @section Body Background
        @tip Set the background colour for the email body.
      */
      table {
        /*@editable*/background-color: #eeeeee !important;
      }</style>
  </head>
<body>
  <!-- An HTML comment to preserve -->
  <table class="w-full" cellpadding="0" cellspacing="0" role="presentation">
    <tr>
      <td class="preserve-it text-center">Test</td>
    </tr>
  </table>
</body>
</html>
